builtin/providers/aws/hosted_zones.go

```diff
@@ -0,0 +1,19 @@
+package aws
+
+// http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_website_region_endpoints
+var hostedZoneIDsMap = map[string]string{
+	"us-east-1":      "Z3AQBSTGFYJSTF",
+	"us-west-2":      "Z3BJ6K6RIION7M",
+	"us-west-1":      "Z2F56UZL2M1ACD",
+	"eu-west-1":      "Z1BKCTXD74EZPE",
+	"central-1":      "Z21DNDUVLTQW6Q",
+	"ap-southeast-1": "Z3O0J2DXBE1FTB",
+	"ap-southeast-2": "Z1WCIGYICN2BYD",
+	"ap-northeast-1": "Z2M4EHUR26P7ZW",
+	"sa-east-1":      "Z7KQH4QJS55SO",
+	"us-gov-west-1":  "Z31GFT0UA1I2HV",
+}
```

Comment:
@justincampbell I was asking our contact at AWS about this, linking him directly to the related PR at aws-sdk-go.
He was assuring me that if these zone IDs were to change, it would most likely mean changing the domain for S3 websites and/or CloudFront global domain (unlikely to happen in one day, if ever) and more importantly if that ever happens, they would give more than enough time to users to adopt this change.

He also said they _may_ expose it via API _at some point_, but from my experience, it could mean 6 months, but also 6 years, or even never. There isn't any specific ETA nor guarantee.